prow/config/periodics.yaml

```diff
@@ -76,4 +76,38 @@ periodics:
           memory: 4Gi
     volumes:
     - emptyDir: {}
-      name: varlibdocker
\ No newline at end of file
+      name: varlibdocker
+
+- name: merge-upstream-istio-2.4
```

Comment:
NIT: do we need to have `-2.4` suffix here? I wonder if we foresee having such a job for multiple branches. If not I would suggest removing it, otherwise, we will need to do small changes every single time there's a new dev branch set. We define the target branch as a parameter anyway.

Reply:
Yeah, eventually we could have multiple sync jobs (1.16, 1.17, master, etc).